aspnetcore/signalr/security.md

```diff
@@ -53,6 +53,58 @@ public void Configure(IApplicationBuilder app)
 > [!NOTE]
 > SignalR is not compatible with the built-in CORS feature in Azure App Service.
 
+### WebSocket Origin Restriction
```

Comment:
Shouldn't this section be on the [CORS](https://docs.microsoft.com/en-us/aspnet/core/security/cors?view=aspnetcore-2.1) page and we just reference it in the SignalR docs?

Reply:
>It's not CORS! :)

Fine, but people assuming it is will be confused. Maybe CORS doc should reference this when it's in WebSockets